### ISSUE TITLE
RSPB acronym

### ISSUE THREAD

**Issue by jbuller** [ministryofjustice/record-a-goose-sighting#9]
'RSPB' is not explained which breaks WCAG 3.1.4 https://www.w3.org/TR/2008/REC-WCAG20-20081211/#meaning-located 
Unless we judge that:
1. linking to the Royal Society for the Protection of Birds website is "linking to a definition". But the expansion is only given at the bottom of the homepage
2. The acronym is well known and the charity actually routinely uses it in preference to the expanded version so that is its name.
Perhaps this should be noted this in the answers.

**Comment by BNewing** [comment]
Ah! Thank you for raising this, I hadn't known that!

Hmm. Similarly to the flashing image issue (#7) - we could either add this in, or remove it. Is this too much detail about WCAG for this training exercise? Or is it useful for highlighting the nuances of WCAG? I see pros and cons to both approach - what do you think?

**Comment by injms** [comment]
It would seem that they seem to be using the acronym in preference to the full name in their writing - the only regular mention of the expanded name I can find on their website is in the footer, alongside the legally required charity numbers. 

I wonder if that fulfils the criteria that they prefer 'RSPB' over 'The Royal Society for the Protection of Birds'?

**Comment by BNewing** [comment]
Thank you for the thoughts on this! It's been really interesting. For now, I've removed it from the answers page.

Maybe it'd be nice to have a section of 'Gotchas' though, where we talk about things that look like they might be issues, but actually aren't. For example, tab behaviour not working by default in Firefox + Safari, and radio button behaviour? This could live in that section :)

**Comment by jbuller** [comment]
To add/keep this issue but avoid difficult nuances of this particular example, you could link instead to the page of another organisation that uses an acronym but that is not so well known and more commonly use the full expanded version.

For example "BTO guide to identifying Grey Geese" > https://www.bto.org/develop-your-skills/bird-identification/videos/grey-geese
